Source/DafnyCore/Verifier/BoogieGenerator.cs

```diff
@@ -143,7 +144,7 @@ public class TranslatorFlags {
       Contract.Invariant(codeContext == null || codeContext.EnclosingModule == currentModule);
     }
 
-    [Pure]
+    [System.Diagnostics.Contracts.Pure]
```

Comment:
Are these the same attribute they were before? I kind of like the unqualified name better, though that's a matter of taste.

Reply:
I believe so, and I think this change was probably applied automatically as a result of the `JetBrains.Annotations` import. I'll undo it in a patch.

Reply:
(I changed my mind on this - see https://github.com/dafny-lang/dafny/pull/5455#discussion_r1605604633)